test/test.js

```diff
@@ -1,20 +1,88 @@
-var expect = require('chai').expect;
-var dirtree = require('../lib/directory-tree');
+var expect = require('chai').expect,
+        dirtree = require('../lib/directory-tree'),
```

Comment:
You probably tried to align these, but it didn't work out. I suggest you keep the old format.